#pragma once
#include "ros/ros.h"
#include "vector"
#include <std_srvs/Empty.h>

#include "steering/Odrive_command.h"
#include "steering/Desired_angle.h"
#include "steering/Steering_loopAction.h"

#include <actionlib/client/simple_action_client.h>
#include "../Odrive/Odrive.hpp"

namespace Communication{


class roscom{
    private:
    
  
};
}